site/public/css/polls.css

```diff
@@ -116,6 +116,25 @@ form {
     margin-left: 0.5em;
 }
 
+#timer-inputs {
+    display: inline-block;
+}
+
+.poll-timer {
+    display: inline-block;
+    text-align: center;
+    width: 4em !important;
```

Comment:
Why is the `!important` necessary here?

Reply:
Removing it messes up the UI, 
Not sure why but I think removing it makes the CSS styling get overridden by something else.